### DISCUSSION TITLE
用 R Markdown 做 PPT | /cn/2018/04/r-markdown-ppt/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1109] [Comments]
https://yihui.org/cn/2018/04/r-markdown-ppt/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
应该还做不了，因为用 Markdown 很难实现细节调整。但可以一试啊，看看能做到什么程度。

> ——原帖发布于 2018-04-26 14:34:51

**Comment by think-crow** [comment]
这本书点赞：[blogdown：使用 R Markdown 创建网站](https://bookdown.org/yihui/blogdown/)！

**Reply by yihui** [reply]
这书很一般，何赞之有？

**Reply by think-crow** [reply]
大哥，你的回复让我挺难过，不知道为什么对这本书评价这么低！

其实国内的大多数人都忽略了网站的价值（除了部分技术人员，普通人很难有自己的网站，大学之下的学生几乎团灭百分之八十），而你这边不仅做出来，而且写成了书，至少到现在还没见到一个国人因网站专门去出一本书！

未来，网站是否应同普通东西一样，每个人都有一个！
仿佛从这本书上看到了未来！

里面的内容步骤还没有实践，但看过你设计的主题，包括这个：[Paged](https://paged.yihui.org/)，及：[博客文章脚注右浮](https://github.com/yihui/yihui.org/discussions/1561)  好像一个惊喜接着一个，就很难再对这本书打低！

**Reply by think-crow** [reply]
而且你也说了，也很赞同：
> I firmly believe in the value of writing.
我对blogdown的兴奋程度甚至超过了bookdown（我之前的项目）。

第一句我觉得谷歌翻译的不准确，就英文复制过来了！（翻译的 坚信写作的价值，但看见value仿佛看到了返回值，可以理解为写作的反馈）

**Reply by yihui** [reply]
在我看来，这书有价值的只有两页：一是[第 2.5 节](https://bookdown.org/yihui/blogdown/templates.html)庖丁解牛式授人以渔，告诉读者如何从地基打造出一个主题；二是[附录 B](https://bookdown.org/yihui/blogdown/website-basics.html) 的网页三件套 HTML/CSS/JS 入门。学会了这两样功夫，就可以甩掉这本书以及 blogdown 了。

你说的网站的价值当然很重要，附录 E 也有提及。这事我[已经说了多年](https://yihui.org/cn/2016/12/student-project/)，在书中再说一遍我自己觉得已经没多大意思了。

然后你说的 paged 主题和脚注右浮那确实是我的得意之作，与 blogdown 并没有太直接的关系。得意之处在于让人知道，代码贵不在多而在精。这些很好用的功能并不需要很复杂的代码就足以实现。

我不仅是对这本书评价低，而是对我过去十几年打造的这一套 R Markdown 评价低。它很有用，但太过臃肿，所以今年我花了几个月从头打造了一套精简版的 litedown。
